projects/demo/src/modules/components/input-files/examples/3/index.html

```diff
@@ -29,6 +29,7 @@ <h4>With link</h4>
         rel="noreferrer noopener"
         state="normal"
         target="_blank"
+        title="File link"
         tuiFile
```

Comment:
`tuiFile` is a component that has file name inside, no need for `title` here

Reply:
fixed